includes/polling/ports.inc.php

```diff
@@ -921,6 +921,9 @@
             if ($device['os'] == 'ios' || $device['os'] == 'iosxe') {
                 include 'ports/cisco-if-extension.inc.php';
             }
+
+            // Do ifOperStatus
+            include 'ports/status.inc.php';
```

Comment:
You haven't included that file here to be reviewed.

Reply:
i thought it is:
includes/html/graphs/port/status.inc.php
bt moving to includes/polling/ports does not help